### PR TITLE
feat(console): icon for custom resources

### PR DESCRIPTION
**Pull request by polamoros**
Resolves https://github.com/winglang/wing/issues/6587

### CODE REVIEW

File: apps/wing-console/console/server/src/consoleLogger.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { errorMessage } from "@wingconsole/error-message";
+import type { DisplayInfo } from "@winglang/sdk/lib/core/tree.js";
```
Comment (ainvoner) [inline apps/wing-console/console/server/src/consoleLogger.ts:2]: not needed

**Comment by ainvoner** [comment]
@polamoros metadata icon doesn't change

Screenshot 2024-05-29 at 14 31 48

**Review by ainvoner — CHANGES_REQUESTED** [review]

**Comment by ainvoner** [comment]
@polamoros also - connection icon

**Comment by polamoros** [comment]
> @polamoros also - connection icon
> 
> Screenshot 2024-05-29 at 14 33 16

fixed

image

**Review by ainvoner — APPROVED** [review]

**Pull request merged**